back/app/models/user.rb

```diff
@@ -331,8 +331,8 @@ def anon_first_name
   end
 
   def anon_last_name
-    # Generate a last name based on email in the format of '123456'
-    email.hash.abs.to_s[0, 6]
```

Comment:
Turns out that a hash is different across different servers / interpreters, so replaced with another lightweight implementation